.env

```diff
@@ -6,6 +6,6 @@ SESSION_SECRET=mySecret
 JWT_SECRET=secretKey123456
 JWT_ISSUER=adc-wayfarer.heroku.com
 API_BASE=/api/v1
-
+DATABASE_URL=postgresql://postgres:admin@localhost:5432/wayfarer
```

Comment:
put `.env` file in the `.gitignore`. Your environment variables should not be found in the public space

Reply:
Ok, let me implement this. Thanks